src/app/clusters/on-off-server/on-off-server.h

```diff
@@ -158,6 +158,9 @@ struct OnOffEffect
  */
 void emberAfOnOffClusterLevelControlEffectCallback(chip::EndpointId endpoint, bool newValue);
 
+void emberAfOnOffClusterLevelControlIsEffectActiveCallback(chip::EndpointId endpoint, bool * const isActive,
```

Comment:
Maybe there's a better way than introducing an "ember" callback to determine whether an OnOff effect is currently active?